v1/random.yaml

```diff
@@ -37,7 +37,7 @@ paths:
               status: 303
               headers:
                 cache_control: '{{options.random_cache_control}}'
-                location: '../../{request.params.format}/{title_from_mobileapps.body.items[0].title}'
+                location: '../../page/{request.params.format}/{title_from_mobileapps.body.items[0].title}'
```

Comment:
Since `/page/` is effectively the root here, would be better to just remove one `../` instead.